app/code/core/Mage/Core/Model/Config.php

```diff
@@ -483,15 +483,15 @@ protected function _canUseCacheForInit()
     {
         if (Mage::app()->useCache('config') && $this->_allowCacheForInit) {
             $retries = 10;
-            do {
+            while ($retries--) {
```

Comment:
this reduces the retries by one compared to before the change

Reply:
this was not complete and marked as draft